runtime/manifest.js

```diff
@@ -193,10 +193,12 @@ ${e.message}
     manifest._schemas[schemaItem.name] = new Schema(schemaItem);
   }
   static _processParticle(manifest, particleItem, loader) {
+    assert(particleItem.implFile == null || particleItem.args !== null, "no valid body defined for this particle");
```

Comment:
I think that the matching name of the Particle in 

maybe we could verify this in 
particle Thing in 'thing.js' 
and in the args definition, should be verified in the peg file:
https://github.com/PolymerLabs/arcs/blob/master/runtime/manifest-parser.peg#L87
instead?

but the implFile (and the description) in the ParticleSpec constructor?
WDYT?